Script.js

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Your script.js file shouldn't have any HTML in it. It should look something like this. https://gist.github.com/blisteringherb/fbc6461b4ec5aae4580638835144b53f  

`Script.js` should also be all lowercase letters.